src/factories/createMockLogger.ts

```diff
@@ -27,41 +35,18 @@ export const createMockLogger = (
     return {};
   };
 
-  log.trace = (a, b, c, d, e, f, g, h, i, j) => {
-    log.child({
-      logLevel: logLevels.trace,
-    })(a, b, c, d, e, f, g, h, i, j);
-  };
-
-  log.debug = (a, b, c, d, e, f, g, h, i, j) => {
-    log.child({
-      logLevel: logLevels.debug,
-    })(a, b, c, d, e, f, g, h, i, j);
-  };
-
-  log.info = (a, b, c, d, e, f, g, h, i, j) => {
-    log.child({
-      logLevel: logLevels.info,
-    })(a, b, c, d, e, f, g, h, i, j);
-  };
-
-  log.warn = (a, b, c, d, e, f, g, h, i, j) => {
-    log.child({
-      logLevel: logLevels.warn,
-    })(a, b, c, d, e, f, g, h, i, j);
-  };
-
-  log.error = (a, b, c, d, e, f, g, h, i, j) => {
-    log.child({
-      logLevel: logLevels.error,
-    })(a, b, c, d, e, f, g, h, i, j);
-  };
-
-  log.fatal = (a, b, c, d, e, f, g, h, i, j) => {
-    log.child({
-      logLevel: logLevels.fatal,
-    })(a, b, c, d, e, f, g, h, i, j);
-  };
+  log.debug = createChildLogger(log, logLevels.debug);
+  log.debugOnce = createChildLogger(log, logLevels.debug);
+  log.error = createChildLogger(log, logLevels.error);
+  log.errorOnce = createChildLogger(log, logLevels.error);
+  log.fatal = createChildLogger(log, logLevels.fatal);
+  log.fatalOnce = createChildLogger(log, logLevels.fatal);
```

Comment:
I don't know what the mock logger does exactly but note that it doesn't use `createOnceChildLogger`?

Reply:
It's just a noop when Roarr is disabled.